kimchi/src/circuits/constraints.rs

```diff
@@ -248,11 +251,13 @@ pub enum GateError {
 }
 
 /// Represents an error found when computing the lookup constraint system
-#[derive(Debug)]
+#[derive(Error, Debug)]
 pub enum LookupError {
     /// One of the lookup tables has columns of different lengths
+    #[error("Inconsisten table length")]
     InconsistentTableLength,
     /// The combined lookup table is larger than allowed by the domain size
+    #[error("Lookup table too long")]
```

Comment:
nit: would have been nice to also display the values `length` and `maximum_allowed` in the error message